setup.py

```diff
@@ -86,38 +86,38 @@
     'pillow>=3.4.2,<4',
     'Babel==2.5.3',
     'cryptography==1.7.1',
-    'Django>=1.8,<1.10',
-    'django-bootstrap3>=6.1,<7',
-    'django-countries>=3.3,<4',
-    'django-enumfields>=0.7.2,<0.8',
-    'django-filer>=1.0,<2',
-    'django-filter>=1.0,<1.1',
-    'django-jinja>=1.4,<2',
-    'django-mptt>=0.8.0,<0.9',
-    'django-parler>=1.5,<1.6.3',
-    'django-parler-rest>=1.3a1,<2',
-    'django-polymorphic>=0.8.0,<0.10',
-    'django-registration-redux>=1.2,<1.5',
-    'django-timezone-field>=1.2,<2',
+    'Django>=1.8,<2',
+    'django-bootstrap3>=9,<10',
+    'django-countries>=5,<5.3',
+    'django-enumfields>=0.9,<0.10',
+    'django-filer>=1.3,<2',
+    'django-filter>=1.1,<1.2',
+    'django-jinja>=2.3,<3',
+    'django-mptt>=0.9,<0.10',
+    'django-parler>=1.9,<2',
+    'django-parler-rest>=1.4.2,<2',
+    'django-polymorphic>=0.9.2,<2.1.0',  # For Django 1.8 and 1.9 use version lower 1
+    'django-registration-redux>=1.5,<1.9',
+    'django-timezone-field>=2.1,<3',
     'djangorestframework>=3.5,<3.6',
     'djangorestframework-jwt>=1.9,<2',
-    'django_rest_swagger==2.1.0',
+    'django_rest_swagger>=2.1.0,<2.2',
     'factory-boy==2.7.0',
     'fake-factory>=0.5.0,<0.5.4',
-    'Jinja2>=2.8,<2.9',
-    'jsonfield>=1.0,<2',
-    'keyring==10.2',
-    'keyrings.alt==2.0',
+    'Jinja2==2.8.1',
```

Comment:
Why is jinja pinned to exact version? Doesn't seem right...

Reply:
Yeah shouldn't be needed but can't fix all the issues as part of this task. @suutari-ai it's locked since that is pretty much only version that works with our front (see https://github.com/shuup/shuup/issues/1285).